welkin/models/assessment.py

```diff
@@ -0,0 +1,40 @@
+from welkin.models.base import Collection, Resource
+
+
+class AssessmentRecord(Resource):
+    def create(self):
+        return super().post(
+            f"{self._client.instance}/patients/{self._parent.id}/assessment-records"
+        )
+
+    def get(self):
```

Comment:
We need to make sure all resources can make requests with no parent (`welkin.Assessment(id="foo").get()`). These methods need to accept ID arguments defaulting to None and then search for the needed arguments from the parent only if the ID isn't passed to this method.

Looking back, I notice this is missing in Encounters as well.